python/paddle/trainer/PyDataProvider2.py

```diff
@@ -69,6 +85,25 @@ def __init__(self, dim, seq_type, tp):
         self.seq_type = seq_type
         self.type = tp
 
+    def __repr__(self):
+        """
+        Return a human readable representation like 'InputType(dim=25921, seq_type=SequenceType.NO_SEQUENCE, type=DataType.Dense)'
```

Comment:
This line is too long. please split it.

Reply:
is this ok?
```
    def __repr__(self):
        """
        Return a human readable representation like 
        'InputType(dim=25921,  seq_type=SequenceType.NO_SEQUENCE, type=DataType.Dense)'
```

Reply:
I think each line should be less than 80 characters. This is ok.